### PR TITLE
Various Changes

### PR DESCRIPTION
**Pull request by datashaman**
To get the pkg working in BM.

### CODE REVIEW

File: src/SailthruChannel.php [hunk 1]
```diff
@@ -27,14 +40,22 @@ public function __construct(\Sailthru_Client $sailthru)
      */
     public function send($notifiable, Notification $notification)
     {
-        try {
+        if (!config('services.sailthru.enabled', true)) {
+            Log::info(
+                'Sailthru message sent',
```
Comment (dylanharbour) [inline src/SailthruChannel.php]: Can we change `sent` to `sending` since it has not sent

File: src/SailthruMessage.php [hunk 2]
```diff
@@ -138,6 +138,30 @@ public function template(string $template): SailthruMessage
         return $this;
     }
 
+    /**
+     * @param array $from Associative array with email and name
+     *
+     * @return SailthruMessage
+     */
+    public function from(array $from): SailthruMessage
```
Comment (dylanharbour) [inline src/SailthruMessage.php:146]: Did you manage to test if this does anything useful?

Comment (datashaman) [reply src/SailthruMessage.php:146]: Hmm, lemme try. The template does specify the sender but it can also be blank.

Comment (dylanharbour) [reply src/SailthruMessage.php:146]: If it works, keep it. If not, take it out. Maybe it's also an account specific setting?

**Review by dylanharbour — APPROVED** [review]

**Pull request merged**